bbc/simorgh
# Optimise srcset for ws frontpages

Resolves #2022 

**Overall change:** _Optimize images sizes on frontpages._

**Code changes:**

- _Pass sizes prop to story promo to load the optimal image for different screen sizes._
- _Add new image resolution for frontpages image srcsets._


**To test**
  - You can check that the images being loaded on mobile are optimized by toggling the responsive mode in dev tools, change the device pixel ration _(denoted as DPR)_ to `1`, then check if the images loaded at this breakpoints  `70`, `95`, `144`, `183`, `240`, `320`, `480` and  `624` are optimal for those screen-sizes.

---
